docs/command-line-interface/ds-load-cli/plugins/custom.mdx

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_label: Custom Plugins
+---
+
+# Writing a custom plugin
+
+Plugins have a simple contract with the host `ds-load` command. They can be written in any language.
+
+A set of go packages can be used as a "mini-SDK" for building custom plugins in go. This eliminates much of the "boilerplate" of writing a plugin, including commands like `export-transform`, providing help, and so on.
+
+An example plugin for Hubspot data can be found [here](https://github.com/aserto-demo/ds-load-hubspot).
```

Comment:
Maybe we could add the plugins directory as well https://github.com/aserto-dev/ds-load/tree/main/plugins